documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1581,6 +1601,22 @@ A value of `0` disables this behavior.
 |`0`
 |Specifies the maximum number of milliseconds the oplog/change stream cursor will wait for the server to produce a result before causing an execution timeout exception.
 A value of `0` indicates using the server/driver default wait timeout.
+
+ifdef::community[]
```

Comment:
Thinking about it as we decided to provide incremental snapshotting for MongoDB as TP then it is correct to keep it for product too but mark it as TP.

Reply:
@jpechane Sorry, is this comment directed specifically at these two properties (`signal.data.collection` and `incremental.snapshot.chunk.size`)? That is, should I remove the `[community]` conditional and instead insert a  [product] conditionalized note that designates them as TP?

Reply:
@roldanbob Yes, eactly